dotnet/connPropBuilder.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/*
-* (c) Copyright IBM Corporation 2018
+* (c) Copyright IBM Corporation 2025
```

Comment:
format is date created, date last updated, so should be:

`2018, 2025` 

This will apply to all modified files, with a copyright date.